doc/config.md

```diff
@@ -885,6 +885,9 @@ Such automatically created database entries are cleaned up
 if they stay idle longer than the time specified by the `autodb_idle_timeout`
 parameter.
 
+The special database **pgbouncer** does not need to be included here, it is
```

Comment:
This wording is not strong enough, given that it is an error. I would just leave this out.

Reply:
I will remove it.